### PR TITLE
Pixel workshops for review

### PR DESCRIPTION
**Pull request by Xuan-Wang-Summer**
This is our pixel workshop for review.

### CODE REVIEW

File: content/english/python-pixel/Activities/Activity-1.md [hunk 1]
```diff
@@ -1,6 +1,44 @@
...
+
+### Create your own colorboard!
+
+<hr/>
+Choose your favorite and make a color board to play with! Here are some example colors you can choose from, you can also find your own color.
+
+<img src="../../media/Color-chart.png" width=30%>
+
+{{% notice warning %}}
+**Be careful:** In order to see your image, please click on top left cornor (which says 'Files'), and then click on the image file to see the result.
```
Comment (ozhang22) [inline content/english/python-pixel/Activities/Activity-1.md:36]:
```suggestion
**Be careful:** In order to see your image, please click on top left corner (which says 'Files'), and then click on the image file to see the result.
```

File: content/english/python-pixel/Activities/Activity-1.md [hunk 2]
```diff
@@ -1,6 +1,44 @@
...
+
+Here is an example of how to make a color board with color red, width of 60, length of 30.
+
+```python
+#This is the example for creating a colorboard. 
+from PIL import Image
+img = Image.new('RGB', (60, 30), 'red')
+img.save('pil_red.png')
+```
+![alt text](../../media/whileloopbefore.png "image showing activity one first example")
```
Comment (ozhang22) [inline content/english/python-pixel/Activities/Activity-1.md:22]: Is this the right image?

File: content/english/python-pixel/Activities/Activity-2.md [hunk 3]
```diff
@@ -1,13 +1,19 @@
...
 draft: false
 ---
 
+<hr/>
+
+### Examples for modifying colorboard
+<hr/>
+
+#### Example 1
+make a <b>diagonal</b> on your color board.
```
Comment (ozhang22) [inline content/english/python-pixel/Activities/Activity-2.md:14]:
```suggestion
Make a <b>diagonal</b> on your color board.
```

File: content/english/python-pixel/Activities/Activity-2.md [hunk 4]
```diff
@@ -16,29 +22,47 @@ x = 0
...
-![alt text](../../../media/whileloopbefore.png "image showing while loop first example")
+![alt text](../../media/whileloopbefore.png "image showing while loop first example")
 This is the picture after changing.
-![alt text](../../../media/whileloopafter.png "image showing while loop first example")
+![alt text](../../media/whileloopafter.png "image showing while loop first example")
 <br/><br/>
+<hr/>
+
+#### Example 2
+make a <b>rectangle</b> on your color board.
```
Comment (ozhang22) [inline content/english/python-pixel/Activities/Activity-2.md:38]:
```suggestion
Make a <b>rectangle</b> on your color board.
```

File: content/english/python-basics/conditional-statements-loops/nested-loops.md [hunk 5]
```diff
@@ -2,50 +2,66 @@
...
 ### Nested loops
 To learn *nested loops*, you need to know simple for-loops and while loops. If you have already learnt them, you can skip to the next part. Otherwise, jump to a specific Python section to learn more about these topics:
 
 - <a href="../../../python-basics/loops" target="blank">For-loop and While-loop</a>
-<br/><br/>
+<br/>
 In this section, we will use one loop inside another, which is called *nested loop*.
-<br/><br/>
-#### Nested while loop
+<hr/>
```
Comment (ozhang22) [inline content/english/python-basics/conditional-statements-loops/nested-loops.md:15]: Usually we won't add horizontal lines in our workshops. Please remove.

File: content/english/python-basics/conditional-statements-loops/nested-loops.md [hunk 6]
```diff
@@ -2,50 +2,66 @@
...
  ```
- This is the output of our example.
- ![alt text](../img/nestedex1.png "image showing while loop first example")
- <br/><br/>
-#### Challenge
+This is the output of our example.
+<img src="../../img/nestex1.png" width=30%>
+<br/>
+
+#### <i>Questions</i>
```
Comment (ozhang22) [inline content/english/python-basics/conditional-statements-loops/nested-loops.md:51]: Please do not add html styling (e.g. <i>, <b>) in any of the headers with ###

File: content/english/python-pixel/Activities/Activity-1.md (hunk 1 above)
Comment (ozhang22) [reply content/english/python-pixel/Activities/Activity-1.md:36]: Please fix this elsewhere too.

File: content/english/python-pixel/Activities/Activity-2.md [hunk 7]
```diff
@@ -16,29 +22,47 @@ x = 0
...
+<br/><br/>
+
+<hr/>
+
+### Modifying your own colorboard!
+
+<hr/>
+
+{{% notice warning %}}
+**Be careful:** In order to see your image, please click on top left cornor (which says 'Files'), and then click on the image file to see the result.
```
Comment (ozhang22) [inline content/english/python-pixel/Activities/Activity-2.md:65]:
```suggestion
**Be careful:** In order to see your image, please click on top left corner (which says 'Files'), and then click on the image file to see the result.
```

File: content/english/python-pixel/Activities/activity-10.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
+---
+title: "Activity 10: Chanllege: Make your own image"
```
Comment (ozhang22) [inline content/english/python-pixel/Activities/activity-10.md:2]:
```suggestion
title: "Activity 10: Challenge: Make your own image"
```

File: content/english/python-pixel/Activities/Activity-1.md [hunk 9]
```diff
@@ -1,6 +1,44 @@
 ---
-title: "Create a color board"
+title: "Activity 1: Create a color board"
```
Comment (ozhang22) [inline content/english/python-pixel/Activities/Activity-1.md:2]: Can you rename to activity-1.md? Not sure Github would accept the file rename though.

File: content/english/python-pixel/Activities/activity-10.md [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+date: 2020-09-08T00:00:00Z
+weight: 8
```
Comment (ozhang22) [inline content/english/python-pixel/Activities/activity-10.md:4]:
```suggestion
weight: 10
```

File: content/english/python-pixel/Activities/activity-4.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+img.save("Mycat.png")
+```
+Wow! This is our cat after the blue filter.
+<img src="../../media/bluefiltercat.png" width=50%>
+<br/><br/>
+
+#### Challenge
+Following the example above, try to create your own filters with different colors.
+{{% notice warning %}}
+**Be careful:** In order to see your image, please click on top left cornor (which says 'Files'), and then click on the image file to see the result.
```
Comment (ozhang22) [inline content/english/python-pixel/Activities/activity-4.md:48]:
```suggestion
**Be careful:** In order to see your image, please click on top left corner (which says 'Files'), and then click on the image file to see the result.
```

File: content/english/python-pixel/Activities/activity-8.md [hunk 12]
```diff
@@ -0,0 +1,93 @@
...
+Then create the symmetrical output with respect to horizontal central line and compare it with the previous output. Are they the same?
+{{% /notice %}}
+
+#### Challenge - Flip your cat from left to right
+It's now your turn to flip your cat from left to right! Exciting！
+{{% notice note %}}
+
+Flip the image from left to right is same as creating a symmetrical image with respect to the `vertical central line`.
+<br/>
+Pay attention to variable wdithNew and think carefully about how to get it.
```
Comment (ozhang22) [inline content/english/python-pixel/Activities/activity-8.md:85]:
```suggestion
Pay attention to variable widthNew and think carefully about how to get it.
```

File: content/english/python-pixel/Activities/activity-8.md [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+
+{{% /notice %}}
+
+{{% notice warning %}}
+**Be careful:** In order to see your image, please click on top left corner (which says 'Files'), and then click on the image file to see the result.
```
Comment (ozhang22) [inline content/english/python-pixel/Activities/activity-8.md:90]: I think this note doesn't need to be repeated for each exercise.

File: content/english/python-pixel/colors_and_pixels/access_pixel.md [hunk 14]
```diff
@@ -0,0 +1,89 @@
+---
+title: "Access pixels on image "
```
Comment (ozhang22) [inline content/english/python-pixel/colors_and_pixels/access_pixel.md:2]:
```suggestion
title: "Access pixels on image"
```

File: content/english/python-pixel/Activities/activity-7.md [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+- <a href="../../colors_and_pixels/pixel-on-image" target="blank">Pixel on image</a>
+<br/>
+- <a href="../../../python-basics/conditional-statements-loops" target="blank">Conditions and Loops</a>
+<br/>
+- <a href="../../activities/activity-4" target="blank">Filters</a>
+<br/><br/>
+<hr/>
+
+#### Example - Change background color
+Let us change the background color of nuevo into pink
```
Comment (ozhang22) [inline content/english/python-pixel/Activities/activity-7.md:23]: Our mascot is named nuvi

```suggestion
Let us change the background color of Nuvi into pink
```

File: content/english/python-pixel/Activities/activity-7.md [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+for i in range(width):    
+    for j in range(height):    
+            color = img.getpixel( (i,j) ) #get the color from original image
+            if color == coloryellow: #if it is yellow
+                newimg.putpixel((i,j),colorpink)# change the color to pink
+            else:
+                newimg.putpixel((i,j),color)# else stays the same
+newimg.save("nuevopink.png")
+```
+Wow! This is our new nuevo after changing the background.
```
Comment (ozhang22) [inline content/english/python-pixel/Activities/activity-7.md:46]:
```suggestion
Wow! This is our new Nuvi after changing the background.
```

File: content/english/python-pixel/Activities/activity-7.md [hunk 17]
```diff
@@ -0,0 +1,56 @@
...
+<img src="../../media/nuevopink.png" width=25%>
+<hr/>
+
+#### Challenge - Change background based on your choice
+It's now your turn to change whichever color you want on the nuevo! Exciting！
```
Comment (ozhang22) [inline content/english/python-pixel/Activities/activity-7.md:51]:
```suggestion
It's now your turn to change whichever color you want on Nuvi! Exciting！
```

File: content/english/python-pixel/Activities/activity-4.md [hunk 18]
```diff
@@ -0,0 +1,51 @@
...
+draft: false
+---
+
+<hr/>
+We have already talked about that the image on the screen is made of pixels. Now we can start to learn how to design your own filter on the image. Here are some examples of how to design a basic filter on your image.
+To prepare for the coming section, it is recommended that you review how to open and save an image.
+<br/><br/>
+You can jump to a specific section to review them:
+<br/>
+- <a href="../../../image-manipulation/open-image" target="blank">Open image</a>
```
Comment (ozhang22) [inline content/english/python-pixel/Activities/activity-4.md:14]: These should be also added in the pre-reqs field at the top of the file (see earsketch workshop for examples).

**Review by ozhang22 — APPROVED** [review]
Will approve for now. Let's review further once this is pushed.

**Comment by Xuan-Wang-Summer** [comment]
“Pixel workshop for review"

**Pull request merged**